code/_onclick/hud/new_player.dm

```diff
@@ -14,7 +15,10 @@
 		var/atom/movable/screen/lobby/lobbyscreen = new button_type()
 		lobbyscreen.SlowInit()
 		lobbyscreen.hud = src
-		static_inventory += lobbyscreen
+		if(istype(lobbyscreen, /atom/movable/screen/lobby/button/collapse))
+			static_inventory += lobbyscreen
+		else
+			toggleable_inventory += lobbyscreen
```

Comment:
Surely this should be a variable on `/atom/movable/screen/lobby` that determines whether it can be hidden or will always show up

Would make it easy when adding future lobby features

Reply:
was meaning to do that yes